ansible/roles/test/files/ptftests/arista.py

```diff
@@ -157,11 +158,11 @@ def run(self):
             log_output = self.do_cmd("show log | begin %s" % log_first_line)
             log_lines = log_output.split("\r\n")[1:-1]
             log_data = self.parse_logs(log_lines)
-            if len(log_data) != 0:
+            if len(log_data) > 1:
                 break
             time.sleep(1) # wait until logs are populated
 
-        if len(log_data) == 0:
+        if len(log_data) < 2:
```

Comment:
why? another magic constant

Reply:
same reason as above. I will put some comments before the check

Reply:
The same here

Reply:
fixed this